subcommand/webhook-cert-manager/command_test.go

```diff
@@ -0,0 +1,275 @@
+	webhookConfigTwo := "webhookTwo"
+
+	caBundleOne := []byte("bootstrapped-CA-one")
+	caBundleTwo := []byte("bootstrapped-CA-two")
+
+	secret1 := &v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: secretOne,
+		},
+		StringData: map[string]string{
+			v1.TLSCertKey:       "cert-1",
+			v1.TLSPrivateKeyKey: "private-key-1",
+		},
+		Type: v1.SecretTypeTLS,
+	}
+	secret2 := &v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: secretTwo,
+		},
+		StringData: map[string]string{
+			v1.TLSCertKey:       "cert-2",
+			v1.TLSPrivateKeyKey: "private-key-2",
+		},
+		Type: v1.SecretTypeTLS,
+	}
+
+	webhookOne := &admissionv1beta1.MutatingWebhookConfiguration{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: webhookConfigOne,
+		},
+		Webhooks: []admissionv1beta1.MutatingWebhook{
+			{
+				Name: "webhookOne-under-test",
+				ClientConfig: admissionv1beta1.WebhookClientConfig{
+					CABundle: caBundleOne,
+				},
+			},
+		},
+	}
+	webhookTwo := &admissionv1beta1.MutatingWebhookConfiguration{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: webhookConfigTwo,
+		},
+		Webhooks: []admissionv1beta1.MutatingWebhook{
+			{
+				Name: "webhookOne-under-test",
+				ClientConfig: admissionv1beta1.WebhookClientConfig{
+					CABundle: caBundleTwo,
+				},
+			},
+		},
+	}
+
+	k8s := fake.NewSimpleClientset(webhookOne, webhookTwo, secret1, secret2)
+	ui := cli.NewMockUi()
+	cmd := Command{
+		UI:        ui,
+		clientset: k8s,
+	}
+	cmd.init()
+
+	file, err := ioutil.TempFile("", "config.json")
+	require.NoError(t, err)
+	defer os.Remove(file.Name())
+
+	_, err = file.Write([]byte(configFile))
+	require.NoError(t, err)
+
+	exitCh := runCommandAsynchronously(&cmd, []string{
+		"-config-file", file.Name(),
+	})
+	defer stopCommand(t, &cmd, exitCh)
+
+	timer := &retry.Timer{Timeout: 10 * time.Second, Wait: 500 * time.Millisecond}
+	retry.RunWith(timer, t, func(r *retry.R) {
+		secret1, err := k8s.CoreV1().Secrets("default").Get(context.TODO(), secretOne, metav1.GetOptions{})
+		require.NoError(r, err)
+		require.NotEqual(r, secret1.Data[v1.TLSCertKey], []byte("cert-1"))
+		require.NotEqual(r, secret1.Data[v1.TLSPrivateKeyKey], []byte("private-key-1"))
+
+		secret2, err := k8s.CoreV1().Secrets("default").Get(context.TODO(), secretTwo, metav1.GetOptions{})
+		require.NoError(r, err)
+		require.NotEqual(r, secret2.Data[v1.TLSCertKey], []byte("cert-2"))
+		require.NotEqual(r, secret2.Data[v1.TLSPrivateKeyKey], []byte("private-key-2"))
+
+		webhookConfig1, err := k8s.AdmissionregistrationV1beta1().MutatingWebhookConfigurations().Get(context.TODO(), webhookConfigOne, metav1.GetOptions{})
+		require.NoError(t, err)
+		require.NotEqual(r, webhookConfig1.Webhooks[0].ClientConfig.CABundle, caBundleOne)
+
+		webhookConfig2, err := k8s.AdmissionregistrationV1beta1().MutatingWebhookConfigurations().Get(context.TODO(), webhookConfigTwo, metav1.GetOptions{})
+		require.NoError(t, err)
+		require.NotEqual(r, webhookConfig2.Webhooks[0].ClientConfig.CABundle, caBundleTwo)
```

Comment:
```suggestion
		require.NotEqual(r, webhookConfig2.Webhooks[1].ClientConfig.CABundle, caBundleTwo)
```

Reply:
I think this one should be `[0]`, but I think it makes sense to have a webhook-config with multiple webhooks in it to test that every webhook has its CA updated.